src/SamplesApp/SamplesApp.Wasm.UITests/package.json

```diff
@@ -15,6 +15,6 @@
   },
   "dependencies": {
     "puppeteer": "^1.13.0",
-    "chromedriver": "^74.0.0"
+    "chromedriver": "^76.0.0"
```

Comment:
Must ensure it's compatible with the docker image in CI

Reply:
Well as the test are running on the server with this config since a month now ... I would assume it's compatible :)

Reply:
It's automatically used, as the path does not change per version. So that's fine :)